src/twisted/conch/ssh/_kex.py

```diff
@@ -92,6 +92,12 @@ class _ECDH256:
     """
     Elliptic Curve Key Exchange with SHA-256 as HASH. Defined in
     RFC 5656.
+
+    Note that C{ecdh-sha2-nistp256} takes priority over nistp384 or nistp512.
+    This is the same priority from OpenSSH.
+
+    C{ecdh-sha2-nistp256} is considered preety good cryptography.
```

Comment:
```suggestion
    C{ecdh-sha2-nistp256} is considered pretty good cryptography.
```

by who? is this a Pretty Good Privacy (PGP) reference?

Reply:
sorry. I will fix the typo as a drive by in another PR.

by who?  Alex https://github.com/twisted/twisted/pull/1602#issuecomment-869056649

thanks